docs/content/concepts/ops-jobs-graphs/jobs.mdx

```diff
@@ -22,53 +22,24 @@ Jobs can be launched in a few different ways:
-| <PyObject object="JobDefinition" />    | A job definition. Jobs are the main unit of execution and monitoring in Dagster. Typically constructed using the <PyObject object="job" decorator /> decorator. |
-|                                        |                                                                                                                                                                 |
+| Name                                | Description                                                                                                                                                     |
+| ----------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| <PyObject object="job" decorator /> | The decorator used to define a job.                                                                                                                             |
+| <PyObject object="JobDefinition" /> | A job definition. Jobs are the main unit of execution and monitoring in Dagster. Typically constructed using the <PyObject object="job" decorator /> decorator. |
+|                                     |                                                                                                                                                                 |
 
 ---
 
-## Creating jobs
+## Creating jobs directly from ops
 
-Jobs can be created in several ways:
+Jobs can be created in several ways. It can be built from software-defined assets or directly from ops.
 
-- [From software-defined assets](#from-software-defined-assets)
-- [Directly from ops](#directly-from-ops)
+<Note>
+  If you are looking for ways to create a job from software-defined assets,
+  visit <a href="/concepts/assets/asset-jobs">Asset jobs</a>.
+</Note>
 
-### From software-defined assets
-
-Asset jobs materialize a fixed set of assets each time they run. Additionally, multiple jobs can target overlapping sets of assets:
-
-```python file=/concepts/assets/build_job.py startafter=start_marker endbefore=end_marker
-from dagster import Definitions, asset, define_asset_job
-
-
-@asset
-def asset1():
-    return [1, 2, 3]
-
-
-@asset
-def asset2(asset1):
-    return asset1 + [4]
-
-
-all_assets_job = define_asset_job(name="all_assets_job")
-asset1_job = define_asset_job(name="asset1_job", selection="asset1")
-
-
-defs = Definitions(
-    assets=[asset1, asset2],
-    jobs=[all_assets_job, asset1_job],
-)
-```
-
-Unlike jobs created using the <PyObject object="job"/> decorator where you explicitly define the dependencies when you create the job, the topology of an asset-based job is based on the [assets](/concepts/assets/software-defined-assets) and their dependencies.
-
-### Directly from ops
+An op job can be created in the following ways:
 
 - [Using the @job decorator](#using-the-job-decorator)
 - [From a graph](#from-a-graph)
```

Comment:
I can't comment on some of the things I see since they're not technically your changes, but if you don't mind addressing these:

- [ ] - Line 49 `job` is missing the decorator. Could you please add it? `<PyObject object="job" decorator />`
- [ ] - The `Using the @job decorator` and `From a graph` headings should be H3s, not H4
- [ ] - Line 125 - `software-defined assets` should be `Software-defined Assets`
- [ ] - Line 135 - This paragraph says `the UI`, but as it's now the first mention of the UI on this page, it should be `the Dagster UI`
- [ ] - Line 167 - Should `op-based jobs` now be `op jobs`?